modules/check_in/app/serializers/check_in/vaos/appointment_serializer.rb

```diff
@@ -25,6 +25,18 @@ class AppointmentSerializer
       attribute :facilityPhoneMain do |object|
         object.dig(:facility, :phone, :main)
       end
+
+      attribute :serviceName do |object|
```

Comment:
should we prepend a `clinic` before these attributes to be explicit that this is clinic data?

Reply:
Sure Gaurav, that makes sense. Please find the updates in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)